.github/workflows/sync.yml

```diff
@@ -3,15 +3,22 @@ name: Sync-Mainnet
 on:
   pull_request_review:
     types: [ submitted ]
+  schedule:
+    - cron: '0 0 * * *'
 
 jobs:
   build_ckb:
     name: Build CKB
-    runs-on: [ self-hosted, Linux, X64, building ]
+    runs-on: ubuntu-latest
     if: |
-      github.repository_owner == 'nervosnetwork' &&
-      contains(github.event.review.body, '#sync-mainnet') &&
-      contains(fromJson( '[ "janx", "doitian", "quake", "xxuejie", "zhangsoledad", "jjyr", "TheWaWaR", "driftluo", "keroro520", "yangby-cryptape", "liya2017" ]' ), github.actor)
+      (
+        github.event_name == 'pull_request_review' &&
+        contains(github.event.review.body, '#sync-mainnet') &&
+        contains(fromJson('[ "janx", "doitian", "quake", "xxuejie", "zhangsoledad", "jjyr", "TheWaWaR", "driftluo", "keroro520", "yangby-cryptape", "liya2017" ]'), github.actor)
```

Comment:
I find a way to remove this ugly code, but a little complex, I will pull request after your PR merged. 😄